<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventEditDialog</class>
 <widget class="QDialog" name="EventEditDialog">
  <layout class="QGridLayout" name="dialogLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="mainArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum"/>
     </property>
     <widget class="QWidget" name="mainWidget">
      <layout class="QVBoxLayout" name="mainLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QGridLayout" name="summaryLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="summaryLabel">
           <property name="text">
            <string>Title</string>
           </property>
           <property name="indent">
            <number>8</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="summaryEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="locationLabel">
           <property name="text">
            <string>Where</string>
           </property>
           <property name="indent">
            <number>8</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="locationEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="allDayBox">
         <property name="text">
          <string>All day</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="timeLayout">
         <item row="0" column="0">
          <widget class="QMaemo5ValueButton" name="fromDateButton">
           <property name="text">
            <string>From</string>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>2</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QMaemo5ValueButton" name="fromTimeButton">
           <property name="valueLayout">
            <enum>QMaemo5ValueButton::ValueBesideText</enum>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QMaemo5ValueButton" name="toDateButton">
           <property name="text">
            <string>To</string>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>2</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QMaemo5ValueButton" name="toTimeButton">
           <property name="valueLayout">
            <enum>QMaemo5ValueButton::ValueBesideText</enum>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QMaemo5ValueButton" name="repeatButton">
         <property name="text">
          <string>Repeat</string>
         </property>
         <property name="valueLayout">
          <enum>QMaemo5ValueButton::ValueBesideText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QMaemo5ValueButton" name="untilButton">
         <property name="text">
          <string>Until</string>
         </property>
         <property name="valueLayout">
          <enum>QMaemo5ValueButton::ValueBesideText</enum>
         </property>
         <property name="visible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QMaemo5ValueButton" name="calendarButton">
         <property name="text">
          <string>Calendar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QMaemo5ValueButton" name="alarmButton">
         <property name="text">
          <string>Alarm</string>
         </property>
         <property name="valueLayout">
          <enum>QMaemo5ValueButton::ValueBesideText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ExpandingTextEdit" name="descriptionEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExpandingTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ExpandingTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
